app/uk/gov/hmrc/hmrcemailrenderer/templates/mandate/mandateAgentSelfAuthAdds.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, params("service").toString + " update") {
+
```

Comment:
Ok, if you don't have information on the recipient that's fine.
You might want to add the department instead of a generic "From HMRC", if you think that would help the user to understand who he's receiving it from... it's up to you, but it's a nice to have bit.